src/components/ContactList.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ContactItem from './ContactItem';
+
+const ContactList = props => (
+  <ul className="contact_list">
+    {
+      props.list.map(el => <ContactItem contact={el} key={el.name} />)
```

Comment:
You should add id field for each contact and use it as key

Reply:
Added id field to each contact and use it as a key.